### ISSUE TITLE
Used colors of output is unusable with dark mode

### ISSUE THREAD

**Issue by schidlo** [psss/tmt#38]
I run 'tmt' command in main directory of this git repo and it was very hard to read the output because the red color is too dark for terminal in dark mode (screenshot attached).
![Screenshot from 2019-11-07 10-52-09](https://user-images.githubusercontent.com/914788/68378607-affda780-014c-11ea-81dd-41c7a6693ca7.png)

**Comment by t184256** [comment]
I believe that when `tmt` uses `click` (and `click` uses `colorama`), it's not using 256 color or truecolor, it uses the base 16 colors. I mean, it doesn't instruct the terminal to output stuff in this particular dark shade of red, it just uses a base 'red' color, which is effectively picked up from your terminal settings. And these are supposed to be readable on your background, and if they are not, it's a terminal misconfiguration.

That being said, configurable color support or at least a way to disable it would be nice.

**Comment by psss** [comment]
Yes, agree with @t184256. It should be possible to fine-tune color in your terminal to make them more readable. Coloring is disabled by default when not attached to a terminal, so `tmt | cat` can be used as a quick workaround. We should also make this easily configurable.

**Comment by RHEmployee** [comment]
Hello. Do we have any idea how to configure tmt via config file or something similar?

If is still free, I would like to take it and maybe search for another similar issues which needs some type of configuration on tmt side.

**Comment by RHEmployee** [comment]
Ok. Looks like config approach is described here:
https://github.com/psss/tmt/issues/84

**Comment by psss** [comment]
> I mean, it doesn't instruct the terminal to output stuff in this particular dark shade of red, it just uses a base 'red' color, which is effectively picked up from your terminal settings. And these are supposed to be readable on your background, and if they are not, it's a terminal misconfiguration.

This is a very good point. @schidlo, can you confirm that adjusting your terminal color palette fixes the issue?

> That being said, configurable color support or at least a way to disable it would be nice.

Color disabling can be done using the `NO_COLOR` variable. See `man tmt` for more info.

**Comment by RHEmployee** [comment]
Hi.

@schidlo just pinging around. Is your issue solved?

**Comment by happz** [comment]
I'd like to work on this: one of the recent patches merged basic support for CLI themes. As of now, only used for rendering plugin docs on the command line, but I plan to spread them into other places, including basic colorization of logging and output.
